samples/SocialSample/Startup.cs

```diff
@@ -135,16 +135,16 @@ 127.0.0.1 MsSecSample.localhost.this
                 options.ClaimsIssuer = "OAuth2-Github";
                 options.SaveTokensAsClaims = false;
                 // Retrieving user information is unique to each provider.
-                options.Notifications = new OAuthAuthenticationNotifications
+                options.Events = new OAuthAuthenticationEvents
```

Comment:
No, all the things include Authentication. See OAuthAuthenticationOptions, etc...
Also https://github.com/aspnet/Security/pull/391